### ISSUE TITLE
Unexpected osrelease grain

### ISSUE THREAD

**Issue by yermulnik** [saltstack/salt#27268]
FYI:
```
10:17:28 [root#jail_centos][j:0][~]> tail -1 /var/log/salt/minion
2015-09-21 10:15:25,110 [salt.loaded.int.module.yumpkg][WARNING ][17715] Unexpected osrelease grain '7.1.1503', please report this
10:18:15 [root#jail_centos][j:0][~]> cat /etc/centos-release
CentOS Linux release 7.1.1503 (Core)
10:18:26 [root#jail_centos][j:0][~]> salt-minion --version
salt-minion 2015.8.0 (Beryllium)
```

**Comment by lorengordon** [comment]
Yup, reported earlier in https://github.com/saltstack/salt/issues/27065. @jfindlay fixed it in the 2015.8 branch, so it should be good in the next patch release.

**Comment by jfindlay** [comment]
@yermulnik, thanks for the report.  This is a duplicate of the issue @lorengordon mentioned.  You are welcome to comment there if you have any further input.
